docs/userGuide/syntax/lists.md

```diff
@@ -109,3 +109,126 @@ first number
 - [x] Item 4
 - ( ) Item 5
 </div>
+
+****Customizing the list appearance:****
+<box type="tip" seamless>
+The list icons can be customized by adding an {icon="..."} at the end of a list item or a list itself.
+</box>
+
+
+<div id="main-example-markbind">
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+* item 1 {icon="x"}
+* item 2 {icon="heavy_check_mark"}
+* item 3
+* item 4 {icon="/images/deer.jpg"}
+  
+{icon="fas fa-check"}
+</variable>
+</include>
+
+</div>
+
+<box type="warning">
+
+1. Item-level specifications should only be attached to the list heading or list item, not the list content.
+2. A blank line preceding the content is crucial.
+3. It's important to insert a blank line before list-level specifications.
```

Comment:
Minor: Recommended to use `1. ` for all items in a numbered list so that it is easy to insert items later.
e.g.,
```
1. aaa
1. bbb
1. ccc
```